ykrt/src/compile/jitc_yk/codegen/x86_64.rs

```diff
@@ -343,6 +342,20 @@ impl<'a> X64CodeGen<'a> {
         }
     }
 
+    fn codegen_lookupglobal_instr(
+        &mut self,
+        inst_idx: jit_ir::InstrIdx,
+        inst: &jit_ir::LookupGlobalInstruction,
+    ) {
+        let decl = inst.decl(self.jit_mod);
+        if decl.is_threadlocal() {
+            todo!();
+        }
+        let sym_addr = symbol_vaddr(&CString::new(decl.name()).unwrap()).unwrap();
+        dynasm!(self.asm ; mov Rq(WR0.code()), QWORD i64::try_from(sym_addr).unwrap());
```

Comment:
I think the unwrap is safe, but we should comment why.

Reply:
[7f1ac54](https://github.com/ykjit/yk/pull/1024/commits/7f1ac545c4c0eea6d29789638c1720c4c95d57fa)